src/api/sqs/index.ts

```
@@ -22,6 +22,7 @@ import {
 import { ValidationError } from "../../common/errors/index.js";
 import { RunEnvironment } from "../../common/roles.js";
 import { environmentConfig } from "../../common/config.js";
+import { createOrgGithubTeamHandler } from "./handlers/createOrgGithubTeam.js";
```

ESLint

[error] Unexpected use of file extension "js" for "./handlers/createOrgGithubTeam.js"

(import/extensions)